taxonomy-editor-frontend/src/pages/startproject/index.tsx

```diff
@@ -68,69 +68,73 @@ const StartProject = () => {
           Start a project
         </Typography>
-        <Stack sx={{ mt: 4, mb: 4 }} direction="row" alignItems="center">
-          <Typography sx={{ mr: 4 }} variant="h5">
-            Taxonomy Name
-          </Typography>
-          <Autocomplete
-            sx={{ width: 265 }}
-            options={TAXONOMY_NAMES}
-            onChange={(e, selectedTaxonomy) => {
-              if (selectedTaxonomy)
-                setTaxonomyName(toSnakeCase(selectedTaxonomy));
-              else setTaxonomyName(null);
-            }}
-            renderInput={(params) => <TextField {...params} />}
-          ></Autocomplete>
-        </Stack>
-        <Stack direction="row" alignItems="center">
-          <Typography sx={{ mr: 8 }} variant="h5">
-            Branch Name
-          </Typography>
+        <div>
+          <FormControl fullWidth sx={{ width: 265, mt: 4 }}>
+            <InputLabel id="taxonomy-name-label">Taxonomy</InputLabel>
+            <Select
+              labelId="taxonomy-name-label"
+              id="taxonomy-name"
+              value={taxonomyName}
+              label="Taxonomy"
+              onChange={(event) =>
+                setTaxonomyName(event.target.value as string)
+              }
+            >
+              {TAXONOMY_NAMES.map((taxonomyItem) => (
+                <MenuItem value={taxonomyItem} key={taxonomyItem}>
+                  {taxonomyItem}
+                </MenuItem>
+              ))}
+            </Select>
+          </FormControl>
+        </div>
+
+        <div>
           <TextField
             size="small"
-            sx={{ width: 265 }}
+            sx={{ width: 265, mt: 2 }}
             onChange={(event) => {
               setBranchName(event.target.value);
             }}
             value={branchName}
             variant="outlined"
+            label="Branch Name"
           />
-        </Stack>
-        <Stack sx={{ mt: 4 }} direction="row" alignItems="center">
-          <Typography sx={{ mr: 10 }} variant="h5">
-            Description
-          </Typography>
+        </div>
+
+        <div>
           <TextField
-            sx={{ width: 265 }}
+            sx={{ width: 265, mt: 2 }}
             minRows={4}
             multiline
             onChange={(event) => {
               setDescription(event.target.value);
             }}
             value={description}
             variant="outlined"
+            label="Description"
           />
-        </Stack>
-        {/* Button for submitting edits */}
-        <LoadingButton
-          loading={loading}
-          disabled={!branchName || !taxonomyName}
+        </div>
+
+        <Button
+          variant="contained"
+          sx={{ mt: 3 }}
           onClick={handleSubmit}
-          sx={{ mt: 4, width: 130 }}
-          text="Submit"
-        ></LoadingButton>
+          disabled={!branchName || !taxonomyName || loading}
+        >
+          {!!loading ? <CircularProgress size={24} /> : "Submit"}
+        </Button>
```

Comment:
Since we are changing this, can we add text saying "Importing taxonomy.." with the circular progress? There is no indication to the user that the process is actually happening, except the circular progress.

Reply:
done, added the text